docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes/index.md

```diff
@@ -84,10 +84,10 @@ The following list contains role-based access control actions.
 | `featuremgmt.write`                   | n/a                                                                                     | Write feature toggles.                                                                                                                                                                                                    |
 | `folders.permissions:read`            | `folders:*`<br>`folders:uid:*`                                                          | Read permissions for one or more folders and their subfolders.                                                                                                                                                            |
 | `folders.permissions:write`           | `folders:*`<br>`folders:uid:*`                                                          | Update permissions for one or more folders and their subfolders.                                                                                                                                                          |
-| `folders:create`                      | n/a                                                                                     | Create folders in the root level. If granted together with `folders:write`, also allows creating subfolders under all folders that the user can update.                                                                   |
+| `folders:create`                      | `folders:*`<br>`folders:uid:*`<br>`folders:uid:general`                                 | Create folders or subfolders. If granted with scope `folders:uid:general`, it allows to create root level folders. Otherwise, it allows creating subfolders under the specified folders.                                         |
```

Comment:
I see that use of `<br>` tags is pretty consistent throughout this documentation but it's discouraged because "[br](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-br-element) elements must be used only for line breaks that are actually part of the content, as in poems or addresses."

We should generally use `<p>` tags to split paragraphs in tables or other semantically correct HTML. In this case I think `<ol>` makes sense.

I can make the change to use semantic HTML in another PR.

Reply:
Thanks for pointing this out! Do you want to raise a follow-up PR to switch to another tag? I'm also happy to do it if you have enough on your plate :)

Reply:
Already done it, just need to resolve the conflicts :) https://github.com/grafana/grafana/pull/92740

Reply:
So fast!